sdk/src/vote_transaction.rs

```diff
@@ -5,7 +5,8 @@ use crate::pubkey::Pubkey;
 use crate::signature::{Keypair, KeypairUtil};
 use crate::system_instruction::SystemInstruction;
 use crate::system_program;
-use crate::transaction::{Instruction, Transaction};
+use crate::transaction::Instruction;
```

Comment:
Why split these lines?

Reply:
My IDE, I must have removed them both and re imported them at some point.